EveripediaNetwork/ep-ui
# Featured wiki card link

Make entire featured wiki card a clickable link

_Updated the FeaturedWikiCard component to make the entire card a clickable link._

## How should this be tested?

1. Navigate to the homepage.
2. Verify that the entire featured wiki card is clickable.
3. Click on the featured wiki card and ensure it navigates to the correct wiki page.

## Notes or observations

_Ensure that the clickable area covers the entire card and that the navigation works as expected._

## Linked issues

closes https://github.com/EveripediaNetwork/issues/issues/3036
